src/components/MarketTableVirtualized/MarketsTableVirtualized.tsx

```diff
@@ -124,36 +85,40 @@ export const MarketsTableVirtualized: React.FC<MarketsTableVirtualizedProps> = m
       [translate],
     )
 
-    const columns = useMemo<ColumnDef<Asset>[]>(
+    const columns = useMemo<Column<Asset>[]>(
       () => [
         {
-          accessorKey: 'assetId',
-          header: () => <Text translation='dashboard.portfolio.asset' />,
-          cell: ({ row }) => (
-            <AssetCell assetId={row.original.assetId} symbol={row.original.symbol} />
+          id: 'assetId',
+          Header: () => <Text translation='dashboard.portfolio.asset' />,
+          Cell: ({ row }: { row: Row<AssetWithRelatedAssetIds> }) => (
+            <AssetCell
+              assetId={row.original.assetId}
+              symbol={row.original.symbol}
+              isGrouped={Boolean(row.original.isGrouped)}
```

Comment:
```suggestion
              isGrouped={row.original.isGrouped}
```

since `<AssetCell />` is happy with undefined, can do the falsy dancy

Reply:
Not needed as it will be managed by another PR, removed it